quiche/src/recovery/gcongestion/bbr2.rs

```diff
@@ -537,6 +538,7 @@ impl CongestionControl for BBRv2 {
         &mut self, _rtt_updated: bool, prior_in_flight: usize,
         _bytes_in_flight: usize, event_time: Instant, acked_packets: &[Acked],
         lost_packets: &[Lost], least_unacked: u64, _rtt_stats: &RttStats,
+        recovery_stats: &mut RecoveryStats,
```

Comment:
Could the Recovery module provide an accessor for RecoveryStats instead of having it be an output argument for on_congestion_event?

Reply:
Lets discuss in the above comment: https://github.com/cloudflare/quiche/pull/2057#discussion_r2096491363